extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -360,6 +360,11 @@ describes.sandboxed('amp-ad-network-adsense-impl', {}, () => {
         expect(ampAnalyticsElement.innerHTML.indexOf(url)).to.not.equal(-1);
       });
     });
+    it('centers iframe in slot', () => {
```

Comment:
I think it's useful b/c the code conditions on the width and height values.  It's usually best to exercise all paths through a conditional when you can.  It both assures correctness now and will help catch regressions if someone alters that conditional in the future.